.github/workflows/release.yml

```diff
@@ -78,6 +78,7 @@ jobs:
         uses: actions/checkout@v2
       - name: Package
         shell: bash
+        if: matrix.os == 'ubuntu-latest' ||  matrix.os == 'macos-latest' 
```

Comment:
This should be ubuntu-18.04

Reply:
I changed it.

Reply:
@marcbowes, it did not build Ubuntu when I changed it.

Reply:
I changed all 'ubuntu-latest' to 'ubuntu-18.04' and it worked.